2_accept_offer.py

```diff
@@ -93,7 +93,11 @@ def process_offer(offer, audit):
     + locale.format("%.8f", Decimal(offer_currency_quantity) / COIN, True) + " " + offer_currency_code + " for " \
     + locale.format("%.8f", Decimal(ask_currency_quantity) / COIN, True) + " " + ask_currency_code
 
-  # TODO: Prompt the user for whether the trade is acceptable
+  answer = raw_input("Do you want to continue with the trade? 'Y' to continue, 'N' to quit.  ").lower()
+  while answer not in ['y', 'yes', 'n', 'no']:
+    answer = raw_input("Not recognised. 'Y' to continue, 'N' to quit.  ").lower()
+  if answer != 'y' or answer != 'yes':
```

Comment:
Also in hindsight it'd be neater with a few line breaks before the prompt (as the transaction info is printed just above).

How about just kill the entire thing and I will redo it with some actual brain use this time?

Also I think I have got the doctest module figured out now too.



> On 7 Jan 2015, at 22:54, Ross Nicoll <notifications@github.com> wrote:
> 
> In 2_accept_offer.py:
> 
> > @@ -93,7 +93,11 @@ def process_offer(offer, audit):
> >      + locale.format("%.8f", Decimal(offer_currency_quantity) / COIN, True) + " " + offer_currency_code + " for " \
> >      + locale.format("%.8f", Decimal(ask_currency_quantity) / COIN, True) + " " + ask_currency_code
> >  
> > -  # TODO: Prompt the user for whether the trade is acceptable
> > +  answer = raw_input("Do you want to continue with the trade? 'Y' to continue, 'N' to quit.  ").lower()
> > +  while answer not in ['y', 'yes', 'n', 'no']:
> > +    answer = raw_input("Not recognised. 'Y' to continue, 'N' to quit.  ").lower()
> > +  if answer != 'y' or answer != 'yes':
> Shouldn't this be "and"? As is, it wants it to be both 'y' and 'yes' in order to not exit.
> 
> \
> Reply to this email directly or view it on GitHub.
>